securedrop/alembic/versions/faac8092c123_enable_security_pragmas.py

```diff
@@ -0,0 +1,26 @@
+"""enable security pragmas
+
+Revision ID: faac8092c123
+Revises: 15ac9509fc68
+Create Date: 2018-03-31 10:44:26.533395
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'faac8092c123'
+down_revision = '15ac9509fc68'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    conn = op.get_bind()
+    conn.execute(sa.text('PRAGMA secure_delete = ON'))
+    conn.execute(sa.text('PRAGMA auto_vacuum = FULL'))
+
+
+def downgrade():
+    pass
```

Comment:
Why no downgrade?

Btw, as a consistency check for migrations, I think a good practice is to do the upgrade, then a downgrade, and then verify that the operations went smoothly and that one is back where one started. If we don't do this, we should be aware we are taking on technical debt as we won't be able to move between database revisions smoothly if needed

Reply:
This was intentional. From [the docs](https://sqlite.org/pragma.html):

> The default setting for auto-vacuum is 0 or "none", unless the SQLITE_DEFAULT_AUTOVACUUM compile-time option is used. The "none" setting means that auto-vacuum is disabled.

and

> The default setting for secure_delete is determined by the SQLITE_SECURE_DELETE compile-time option and is normally off.

Because I'm not totally sure what the actual state of the DB is before this migration, I'm leaving it alone.